### PR TITLE
Created the feedback(s) app.

### PR DESCRIPTION
**Pull request by jefftriplett**
Added a feedback form. I can make it ajax if needed / wanted. For now we at least have a working app.

### CODE REVIEW

File: feedbacks/models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+class Feedback(models.Model):
+    name = models.CharField(max_length=200)
+    email = models.EmailField()
+    country = models.CharField(max_length=100)
+    feedback_categories = models.ManyToManyField(FeedbackCategory, related_name='feedbacks')
+    issue_type = models.ForeignKey(IssueType, related_name='feedbacks')
+    referral_url = models.URLField(null=True, blank=True)
+    comment = models.TextField()
+    created = models.DateTimeField(default=timezone.now, blank=True)
```
Comment (jacobian) [inline feedbacks/models.py:29]: Everything here except for the comment should be optional - we want to make it as easy as possible to fill out.

**Comment by jacobian** [comment]
Other than that LGTM. We'll want some styling on the form and it probably shuold be hidden or something by default, but that's probably Jason's department.

**Comment by jefftriplett** [comment]
Updated the feedback models to only require comment. I'll wait until Jason has had time to look then I'll merge in Monday.

**Comment by jhogue** [comment]
I will review this weekend. Fast work!

On Fri, Mar 8, 2013 at 4:28 PM, Jeff Triplett notifications@github.comwrote:

> Updated the feedback models to only require comment. I'll wait until Jason
> has had time to look then I'll merge in Monday.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/9#issuecomment-14646469
> .

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by jhogue** [comment]
Fill me in on what the plan was to access this. One of the "fixed" position
tabs on the right or left, or some other trick? I feel like we could put it
inline onto each page below the footer... give it some legitimate space.

On Fri, Mar 8, 2013 at 5:06 PM, Jason Hogue j@projectevolution.com wrote:

> I will review this weekend. Fast work!
> 
> On Fri, Mar 8, 2013 at 4:28 PM, Jeff Triplett notifications@github.comwrote:
> 
> > Updated the feedback models to only require comment. I'll wait until
> > Jason has had time to look then I'll merge in Monday.
> > 
> > —
> > Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/9#issuecomment-14646469
> > .
> 
> ## 
> 
> _J. Hogue_: Design Director at Project Evolution
> t: 888.456.0467 | c: 401.831.1963

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by jacobian** [comment]
Yeah, the idea was to have this on every page, maybe below the fold? I like
the way Everyblock did this: http://note.io/12Bb2Qn.

**Comment by jhogue** [comment]
Yeah, I have an idea as to how to do it. Is this a form that I can drop in
somehow where ever? Seems like I can drop the contents of {% block content
%} in the feedback_form.html anywhere, correct? I think I know how to go
about it if that's true.

On Fri, Mar 8, 2013 at 6:42 PM, Jacob Kaplan-Moss
notifications@github.comwrote:

> Yeah, the idea was to have this on every page, maybe below the fold? I
> like
> the way Everyblock did this: http://note.io/12Bb2Qn.
> 
> On Fri, Mar 8, 2013 at 5:19 PM, J. Hogue notifications@github.com
> wrote:
> 
> > Fill me in on what the plan was to access this. One of the "fixed"
> > position
> > tabs on the right or left, or some other trick? I feel like we could put
> > it
> > inline onto each page below the footer... give it some legitimate space.
> > 
> > On Fri, Mar 8, 2013 at 5:06 PM, Jason Hogue j@projectevolution.com
> > wrote:
> > 
> > > I will review this weekend. Fast work!
> > > 
> > > On Fri, Mar 8, 2013 at 4:28 PM, Jeff Triplett <
> > > notifications@github.com>wrote:
> > > 
> > > > Updated the feedback models to only require comment. I'll wait until
> > > > Jason has had time to look then I'll merge in Monday.
> > > > 
> > > > —
> > > > Reply to this email directly or view it on GitHub<
> > > > https://github.com/proevo/pythondotorg/pull/9#issuecomment-14646469>
> > > > .
> > > 
> > > ## 
> > > 
> > > _J. Hogue_: Design Director at Project Evolution
> > > t: 888.456.0467 | c: 401.831.1963
> > 
> > ## 
> > 
> > _J. Hogue_: Design Director at Project Evolution
> > t: 888.456.0467 | c: 401.831.1963
> > 
> > —
> > Reply to this email directly or view it on GitHub<
> > https://github.com/proevo/pythondotorg/pull/9#issuecomment-14651879>
> > .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/proevo/pythondotorg/pull/9#issuecomment-14652652
> .

## 

_J. Hogue_: Design Director at Project Evolution
t: 888.456.0467 | c: 401.831.1963

**Comment by jhogue** [comment]
I like the Everyblock idea and would like to put this at the bottom of each page. Can someone add the necessary code to the views.py so I can add the form code into the footer of Base.html? I'll style it up from there and add an accordion feature so that is is not always in view. 

Would it be too much to include a field where someone could upload an image, hopefully a screen shot? If not, that's cool... I realize there is no easy way to ensure that it is an image file and not something potentially malicious.

**Comment by jefftriplett** [comment]
I'm going to work on a mini-form + template tag to add a form to the bottom of the footers. Since there is more information, I'll add one that does the full form too and you can decide which one you want.

**Comment by jhogue** [comment]
Looks good. I think I got it all implemented in base.html where I'd like it. Please review for functionality. Good to merge for me when you are ready.

**Pull request merged**